packages/console/src/pages/ApiResourceDetails/index.module.scss

```diff
@@ -1,5 +1,9 @@
 @use '@/scss/underscore' as _;
 
+.permissionPage {
+  height: 100%;
```

Comment:
min-height?

Reply:
Pages with tables will scroll in the table, not on the page.
<img width="1244" alt="image" src="https://user-images.githubusercontent.com/10806653/210921748-b90bd266-0af9-4151-ac73-7cf14928155a.png">